### ISSUE TITLE
Review minimum year threshold

### ISSUE THREAD

**Issue by timrobertson100** [gbif/pipelines#735]
As recorded [in feedback](https://github.com/gbif/portal-feedback/issues/4130), we have real records from the 16th century, such as https://www.gbif.org/occurrence/3804272302

A limit was used to help remove dirty data. I suggest we simply lower the 1600 limit to 1500 
CC @MattBlissett @ahahn-gbif may have an opinion on this?

**Comment by MattBlissett** [comment]
I agree; we will probably let pass a few bad records, but that's better than breaking these good records.

Also #492, and the TDWG Data Quality group which also has 1600: https://github.com/tdwg/bdq/issues/84 (might be copied from us though).

**Comment by ahahn-gbif** [comment]
Agree. We could consider a kind of "year at end of range" flag, but extending the range sounds reasonable

**Comment by sylmorin-gbif** [comment]
Hello,
I would vote for the flag, so we can configure it.
In France, we have some historical data before 1500.
For exemple, from this dataset **Historical distribution of the wolf in France**, we have date in 1439:
https://openobs.mnhn.fr/openobs-hub/occurrences/d1beaa5c-883a-4c14-a4a8-4866e9c5fcf8
(I know, it's not really reliable)
